src/lib.rs

```diff
@@ -344,6 +344,18 @@ where
                 info!("ready: {}", res);
                 Ok(None)
             }
+            XtStatus::Broadcast => {
+                rpc::send_extrinsic(self.url.clone(), jsonreq, result_in);
```

Comment:
you can't call this function because that is hard-coded to wait for `ready`.

Reply:
I will add a new await function